bgpd/bgpd.c

```diff
@@ -1619,6 +1619,20 @@ static bool bgp_peer_conf_if_to_su_update_v6(struct peer *peer,
 	return false;
 }
 
+static bool bgp_peer_iface_to_su_update_config(struct peer *peer,
```

Comment:
We care only about LL addresses in this function, so I suggest renaming it to be more readable and not confusing.

Reply:
bgp_peer_iface_to_su_update_ll_peer() ?